src/createServer.js

```diff
@@ -1,3 +1,55 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { checkValidateRequest } = require('./validateRequest');
+
+const errorsTypes = {
+  notOriginalText: 'Text to convert is required.'
+  + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  notTargetCase: '"toCase" query param is required.'
+  + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  wrongTargetCase: 'This case is not supported.'
+  + ' Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
```

Comment:
Better to use values from cases list, then you'll be able to just list and this string will be updated